data/json_files/release_future_date_change.json

```diff
@@ -0,0 +1,24 @@
+{
+  "uri": "/employmentandlabourmarket/peoplenotinwork/unemployment/datasets/unemploymentbypreviousoccupationunem02",
+  "uri_old": "/an/old/uri",
+  "content_type": "release",
+  "cdid": "ASELECTIONOFNUMBERSANDLETTERS123",
```

Comment:
I think somewhere else, possibly search-api we do validation on cdid and dataset_id:

https://github.com/ONSdigital/dp-search-api/blob/e6a10527f96c88c4bbe2a52e945e7a149127da3b/api/search.go#L138

https://github.com/ONSdigital/dp-search-api/blob/e6a10527f96c88c4bbe2a52e945e7a149127da3b/api/search.go#L415

So will want to match that - really we should do this in the pipeline as well but that's not the focus of this ticket.

Reply:
It looks like the dataset_id already meets the spec (it only has to be more than 1 character long). I've amended the CDID to be 5 characters long as required.